clientconn.go

```diff
@@ -740,6 +741,23 @@ func (cc *ClientConn) resolveNow(o resolver.ResolveNowOption) {
 	go r.resolveNow(o)
 }
 
+// ResetConnectBackoff wakes up all subchannels in transient failure and causes
```

Comment:
Is the main purpose of this function to reset backoff or to wake up addrConns?

Reply:
Both!